zebedee/client_test.go

```diff
@@ -241,15 +246,20 @@ func TestUnitClient(t *testing.T) {
 	})
 
 	Convey("test GetBulletin", t, func() {
-		Convey("returns a bulletin", func() {
-			b, err := cli.GetBulletin(ctx, testAccessToken, "", "bulletin")
+		Convey("returns the latest release of a bulletin", func() {
+			b, err := cli.GetBulletin(ctx, testAccessToken, "", "bulletin-latest-release")
 			So(err, ShouldBeNil)
 			So(b, ShouldNotBeEmpty)
 			So(b.Type, ShouldEqual, "bulletin")
-			So(b.URI, ShouldEqual, "/economy/environmentalaccounts/bulletins/ukenvironmentalaccounts/2015-07-09")
+			So(b.URI, ShouldEqual, "/bulletin/2015-07-09")
 			So(b.RelatedBulletins, ShouldNotBeEmpty)
 			So(len(b.RelatedBulletins), ShouldEqual, 1)
-			So(b.RelatedBulletins[0].URI, ShouldEqual, "/economy/environmentalaccounts/datasets/ukenvironmentalaccountsenergybridging")
+			So(b.RelatedBulletins[0].URI, ShouldEqual, "pageTitle")
+			So(b.RelatedBulletins[0].Title, ShouldEqual, "baby-names: 2017")
+			So(b.Links, ShouldNotBeEmpty)
+			So(len(b.Links), ShouldEqual, 1)
```

Comment:
Would it make sense to have a test (or extend this one) to retrieve multiple links?

This would cover any issues with multiple calls to `GetPageTitle`